helpers/lib_tasks_docker.py

```diff
@@ -396,13 +396,13 @@ def docker_login(ctx, target_registry="aws_ecr.ck"):  # type: ignore
     """
     _ = ctx
     hlitauti.report_task()
-    # No login required as the `helpers` container is accessible on
-    # the public DockerHub registry.
+    # No login required as the `helpers` and `tutorials` images are accessible
```

Comment:
Fine with me, but moving fwd the logic should go in repo_config.yaml (FYI @heanhsok )

The fact that it's a bad design (no offense @dremdem, I've done the same) is that now `//helpers` needs to know about `//tutorials`, which is a violation of the separation between repos.

I would commit this and file an issue about a monster clean up of all these little violations.

Reply:
Filed
https://github.com/causify-ai/helpers/issues/424